apps/fulltextsearch.sh

```diff
@@ -68,13 +68,12 @@ then
 fi
 
 # Check & install docker
-apt update -q4 & spinner_loading
 install_docker
 set_max_count
 mkdir -p "$RORDIR"
-if docker ps -a | grep "$fts_es_name"
+if does_this_docker_exist "$nc_fts"
 then
-    docker stop "$fts_es_name" && docker rm "$fts_es_name" && docker pull "$nc_fts"
+    docker_prune_this "$nc_fts"
```

Comment:
fyi @Ark74 it not removes everything on a new install as well

Reply:
But the issue you reported should not happen on new installs since they have no data volume nor container running.
From what we tested, fts have issues when there is mixed data, 6.x/7.x, so starting from scratch is necessary for this new release.

I've have upgraded from es7.0 to es7.1 with no issues once I had re-indexed everything from scratch.

Reply:
The script does a re-indexing, removing the container, but I'm not sure about the volume. This issue fixes that anyway. The VM is all about automation you know. ;)